linera-core/src/node.rs

```diff
@@ -283,6 +283,10 @@ impl From<ChainError> for NodeError {
             ChainError::ExecutionError(
                 ExecutionError::SystemError(SystemExecutionError::BlobNotFoundOnRead(blob_id)),
                 _,
+            )
+            | ChainError::ExecutionError(
+                ExecutionError::ViewError(ViewError::BlobNotFoundOnRead(blob_id)),
+                _,
```

Comment:
Should we have a single `ChainError::BlobNotFoundOnRead` instead, that is created in both cases?

Reply:
Yes… not sure if that's better. But certainly in a separate PR, if at all.